Composer/packages/client/src/components/ProjectTree/ExpandableNode.tsx

```diff
@@ -53,8 +51,10 @@ export const ExpandableNode = ({
   }
 
   function handleClick(ev: MouseEvent) {
-    if ((ev.target as Element)?.tagName.toLowerCase() === 'summary') {
+    const target = ev.target as Element;
+    if (target.tagName.toLowerCase() === 'li') {
```

Comment:
can target.tagName be null? If so perhaps through a unary null check operator in there or run a null check on target.tagname

Reply:
I thought about that, but if there's an event at all, it's going to have a target, and tagName is a standard way to get at what HTML tag has been clicked on. You can't click on a space where no element is.